website/docs/dev_addon_creation.md

```diff
@@ -275,10 +275,168 @@ If you are curious, here's what happens when you access the api without logging
 
 
 ### Web Action
-:::note TODO
-Write about it and add an example.
+
+Web actions let users trigger AYON Addons CLI actions directly from the AYON server to perform actions on their machines. They run through **shims**, which are automatically set up when users install the AYON launcher version `1.1.0` or higher.
+
+Web actions are added in two steps:
+
+1. **Client-Side CLI Actions**: Develop the addon CLI actions in the client-side code of the addon. More about them in the next section [CLI Interface](dev_addon_creation.md#cli-interface).
```

Comment:
@iLLiCiTiT I vaguely recall you saying that _technically_ it's not a requirement for the web action to go through the launcher CLI - it could potentially do something on the machine without launcher or CLI being involved? Did I misunderstand?

Reply:
As far as I know, this is not implemented yet in the [executor](https://github.com/ynput/ayon-backend/blob/develop/ayon_server/actions/execute.py).
And, will we need to setup shims for other applications we want to run directly from the web action?

---
Personally, I like using launcher CLI as we can add the logic for different operating systems there instead of the web action.

Reply:
Hmm, yes agreed.